BroadcastPickerView.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+import {
+  requireNativeComponent,
+} from 'react-native';
+
+const View = requireNativeComponent('ScreenCapturePickerView');
+
+export default View;
```

Comment:
You can skip the extra variable and just do `export default requireNativeComponent('ScreenCapturePickerView');`

Reply:
done. The file was renamed to ScreenCapturePickerView